### ISSUE TITLE
`is_elevated` conflicts with the "Add Mark to Log" shortcut

### ISSUE THREAD

**Issue by FPtje** [F1bonacc1/process-compose#236]
## Defect

Take the example from the documentation:

```yaml
processes:
  elevated_ls:
    description: "run an elevated process"
    command: "ls -l /root"
    is_elevated: true
    shutdown:
      signal: 9
```

* Try to run the process
* See a little pop up saying you need to enter the sudo password
* Try to type the letter `m`
* See that no letter is actually typed
* Do witness a mark in your log!

The reason for this is that there is a default shortcut for `m`, which is `Add Mark to Log`. I think shortcuts should be temporarily suspended (except for maybe the Escape key) when the password pop up box is open :)

---

Make sure that these boxes are checked before submitting your issue -- thank you!

- [x] Included the relevant configuration snippet
- [ ] Included the relevant process-compose log (log location: `process-compose info`)
  - The log does not show anything relevant
- [ ] Included a [Minimal, Complete, and Verifiable example] (https://stackoverflow.com/help/mcve)
  - The example from documentation is minimal enough.

#### Version of `process-compose`:
Version v1.24.0

#### OS environment:
Ubuntu 22.04

#### Steps or code to reproduce the issue:
See above

#### Expected result:
The letter M being typed, and no mark being added

#### Actual result:

A mark being added, and the letter M not being typed

**Comment by F1bonacc1** [comment]
Thanks for letting me know.
Fixed in v1.24.2

**Comment by FPtje** [comment]
Wow, that's quick! Thanks!
